python-sdk/nuscenes/eval/detection/loaders.py

```diff
@@ -117,6 +127,22 @@ def filter_eval_boxes(nusc, eval_boxes: EvalBoxes, max_dist: dict):
         # Then remove boxes with zero points in them. Eval boxes have -1 points by default.
         eval_boxes.boxes[sample_token] = [box for box in eval_boxes[sample_token] if not box.num_pts == 0]
 
-        # TODO: add bike-rack filtering
+        # Perform bike-rack filtering
+        sample_anns = nusc.get('sample', sample_token)['anns']
+        bikerack_recs = [nusc.get('sample_annotation', ann) for ann in sample_anns if
+                         nusc.get('sample_annotation', ann)['category_name'] == 'static_object.bicycle_rack']
+
+        filtered_boxes = []
+
+        for rec in bikerack_recs:
+            bikerack_box = Box(rec['translation'], rec['size'], Quaternion(rec['rotation']))
+            for box in eval_boxes[sample_token]:
+                if box.detection_name in ['bicycle', 'motorcycle'] and \
```

Comment:
Include motorcycle?

Reply:
According to the instructions https://github.com/nutonomy/nuscenes-devkit/blob/master/instructions.md#bicycle-rack, a bike rack only mentions bicycles. But I know from our review that sometimes motorcycles are parked inside bike racks.